data/netcdf_data.py

```diff
@@ -128,8 +128,8 @@ def find_variable(substring: str, variables: list):
         variable_list = [v.key for v in self.variables]
```

Comment:
Is `variable_list` being used elsewhere below?

Reply:
Doesn't look like it, should I comment it out or remove it?